src/test/java/net/minecraftforge/debug/recipe/recipebook/RecipeBookExtensionTest.java

```diff
@@ -49,24 +47,18 @@
     public static final RegistryObject<MenuType<RecipeBookTestMenu>> RECIPE_BOOK_TEST_MENU_TYPE =
             MENU_TYPE.register("test_recipe_menu", () -> IForgeMenuType.create(RecipeBookTestMenu::new));
 
+    public static final DeferredRegister<RecipeType<?>> RECIPE_TYPE = DeferredRegister.create(ForgeRegistries.RECIPE_TYPES, MOD_ID);
+    public static final RegistryObject<RecipeType<RecipeBookTestRecipe>> RECIPE_BOOK_TEST_RECIPE_TYPE = registerRecipeType("test_recipe");
+
     public RecipeBookExtensionTest()
     {
         if (!ENABLED)
             return;
-        FMLJavaModLoadingContext.get().getModEventBus().addListener(this::commonSetup);
         RECIPE_SERIALIZER.register(FMLJavaModLoadingContext.get().getModEventBus());
         MENU_TYPE.register(FMLJavaModLoadingContext.get().getModEventBus());
         MinecraftForge.EVENT_BUS.addListener(this::onRightClick);
     }
 
-    private void commonSetup(FMLCommonSetupEvent event)
-    {
-        event.enqueueWork(() ->
-        {
-            RecipeType.register(getId("test_recipe").toString());
```

Comment:
Should `RecipeType.register` and similar methods be marked deprecated, to warn modders who are currently calling it to move to `DeferredRegister`s instead (as seen in this test mod)?

Reply:
What similar methods?